ketcher-autotests/tests/utils/selectors/templateModal.ts

```diff
@@ -94,6 +94,57 @@ export enum TemplateLibrary {
   Arabinofuranose = 'Arabinofuranose',
 }
 
+export enum Sugars {
+  TwelveddR = "12ddR___1',2'-Di-Deoxy-Ribose",
+  TwentyFiveR = '25R___2,5-Ribose',
+  ThreeA6 = "3A6___6-amino-hexanol (3' end)",
+}
+
+export async function selectSugar(sugarName: Sugars, page: Page) {
+  await page.getByTestId('RNA-TAB').click();
+  await page.getByTestId('summary-Sugars').click();
+  await page.getByTestId(sugarName.toString()).click();
```

Comment:
Is 'toString()' on sugarName necessary? I thought that, for example, 'Sugars.TwelveddR' would return "12ddR___1',2'-Di-Deoxy-Ribose" as string? Just asking because I'm not sure how it works at TypeScript ;)